doc/projects.rst

```diff
@@ -34,8 +33,8 @@ To create a :guilabel:`Project`, click the :guilabel:`New Project` entry in the
 Using the Project Explorer
 ==========================
 
-Once a :guilabel:`Project` is opened, the :guilabel:`Project Explorer` pane is shown, presenting a tree view of the current :guilabel:`Project` 's files and directories.
-This pane allows you to perform all the same operations as a normal Spyder :doc:`fileexplorer`.
+Once a :guilabel:`Project` is opened, the :guilabel:`Project` pane is shown, presenting a tree view of the current :guilabel:`Project`'s files and directories.
+This pane allows you to perform all the same operations as a regular Spyder :doc:`fileexplorer`.
```

Comment:
```suggestion
This pane allows you to perform all the same operations you can do in the :doc:`fileexplorer` pane.
```

This sentence sounds odd to me, so I rephrased it to be simpler.

Reply:
```suggestion
It allows you to perform all the same operations as Spyder's :doc:`fileexplorer` pane.
```

Fix grammar/phrasing and avoid repetition